### PR TITLE
nikolai.berestevich/fix/derivx logo changed

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-   derivx logo was changed

## Type of change

-   [ x ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ x ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/images/svg/deriv-x/derivx-logo-light.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
```
Comment (yashim-deriv) [inline src/images/svg/deriv-x/derivx-logo-light.svg:1]: Please minify this file. 🙏🏼

File: src/pages/derivx/_hero.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import DHero from 'components/custom/_dhero-2'
 import { size } from 'themes/device'
 import DerivXBG from 'images/svg/deriv-x/triangle-up.svg'
 import DerivXBG2 from 'images/svg/deriv-x/triangle-down.svg'
-import deriv_x_logo from 'images/svg/deriv-x/derivx-logo.svg'
```
Comment (yashim-deriv) [inline src/pages/derivx/_hero.js:7]: SVG imports are treated as components. So the naming should follow components. In this case: `DerivXLogo` would be a more better name

File: src/images/svg/deriv-x/derivx-logo-light.svg [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-        <path d="M0 6a6 6 0 0 1 6-6h20a6 6 0 0 1 6 6v20a6 6 0 0 1-6 6H6a6 6 0 0 1-6-6V6z" fill="#FFF"/>
-        <path d="m6.706 10.5 4.21 5.5-6.75 5.5h6.823a4.279 4.279 0 0 0 4.167-3.316l.565-2.447a4.275 4.275 0 0 0-4.167-5.237H6.706z" fill="url(#a)"/>
-        <path fill="#CC2F3E" d="m10.917 16-3 5.5h-3.75z"/>
-        <path fill="#0E0E0E" d="M20.526 10.5H17.27l2.563 5.5-4.503 5.5h3.258l4.5-5.5zM24.702 10.5h3.295l-3.907 4.726-1.18-2.535zM22.76 21.5l-1.019-2.189 2.076-2.535 2.201 4.724z"/>
-    </g>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+<defs>
+<linearGradient id="a" x1="63.183%" x2="44.398%" y1="-9.799%" y2="101.95%">
+<stop stop-color="#F26447" offset="0"/>
```
Comment (sean-binary) [inline src/images/svg/deriv-x/derivx-logo-light.svg:5]: You can run `svgo` to minify these files.

File: src/pages/derivx/_hero.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import DHero from 'components/custom/_dhero-2'
 import { size } from 'themes/device'
 import DerivXBG from 'images/svg/deriv-x/triangle-up.svg'
 import DerivXBG2 from 'images/svg/deriv-x/triangle-down.svg'
-import deriv_x_logo from 'images/svg/deriv-x/derivx-logo-light.svg'
+import DerivXlogoL from 'images/svg/deriv-x/derivx-logo-light.svg'
```
Comment (sean-binary) [inline src/pages/derivx/_hero.js:7]: Should be `DerivXLogoL` instead, with a capital `L` if we're uppercasing every new word.

**Review by yashim-deriv — APPROVED** [review]
LGTM

**Pull request merged**